Library/Formula/collectd.rb

```diff
@@ -80,4 +81,16 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    begin
+      fork { system sbin/"collectd" }
+      sleep 2
+      assert File.exist?("#{var}/run/collectd.pid")
+    ensure
+      pid = File.read("#{var}/run/collectd.pid").to_i
+      Process.kill("SIGINT", pid)
+      Process.wait(pid)
+    end
```

Comment:
Test is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/30168/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_collectd/